_includes/head.html

```diff
@@ -10,4 +10,7 @@
     <link rel="canonical" href="{{ page.url | replace:'index.html','' | prepend: site.baseurl | prepend: site.url }}">
     <script type="text/javascript" src="{{ "/assets/js/jquery.js" | prepend: site.baseurl }}"></script>
     <script type="text/javascript" src="{{ "/assets/js/materialize.js" | prepend: site.baseurl }}"></script>
+    <link rel="dns-prefetch" href="//matomo.opennic.org">
+    <link rel="dns-prefetch" href="//api.opennic.org">
+    <link rel="preload" href="https://matomo.opennic.org/piwik.js" onload="embedTracker()" type="script" crossorigin>
```

Comment:
I don't think that we want this in the head but rather keep the Piwik code in the end-of-body where it should load after all other assets have been loaded. Can you do some performance comparisons between this being in the head vs. not in the head?

Reply:
The preload attribute just ensures piwik.js is downloaded by the browser for later use by the page (i.e. when the actual Piwik script in the footer is executed), as opposed to the browser downloading the file when it reaches that point in the script. 

It's only supported in Chrome at the moment, although Edge, Safari, and Firefox support is coming. https://caniuse.com/#feat=link-rel-preload. It doesn't affect the window's onload event.